docset/winserver2019-ps/startlayout/Export-StartLayout.md

```diff
@@ -51,6 +51,13 @@ PS C:\> Export-StartLayout -Path "C:\Layouts\Marketing.json"
 
 This command exports the layout of the tiles on the Windows 11 Start menu to a file named Marketing.json in the C:\Layouts folder. Note that JSON is the only supported format for the output file and only the `-Path` parameter is supported.
 
+### Example 3: Export the layout with applications' IDs
+```
+PS C:\> Export-StartLayout -UseDesktopApplicationID  -Path "C:\Layouts\Marketing.json"
+```
+
+This command exports the layout of the tiles on Start menu of a computer running Windows 10 1809 or later to a file named Marketing.json in the C:\Layouts folder. It exports applications' IDs instead of paths to a shortcut links.
```

Comment:
```suggestion
This command exports the layout of the tiles on the Start menu of a computer running Windows 10, version 1809 or later, to a file named Marketing.json in the C:\Layouts folder. It exports the applications' IDs instead of paths to a shortcut link.
```